src/test/scala/me/jeffmay/util/TestInitialization.scala

```diff
@@ -0,0 +1,5 @@
+package me.jeffmay.util
+
+class TestInitialization {
+
+}
```

Comment:
FAIL: This should be removed regardless of whether I use `TestHooks`. There isn't anything to initialize.